src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/AvoidNestedBlocksCheck.java

```diff
@@ -61,6 +61,54 @@
  * the allowInSwitchCase property to true and include all statements
  * of the case in the block.
  * </p>
+ * <ul>
+ * <li>
+ * Property {@code allowInSwitchCase} - Allow nested blocks if they are the
+ * only child of a switch case.
+ * Type is {@code boolean}.
+ * Default value is {@code false}.
+ * </li>
+ * </ul>
+ * <p>
+ * To configure the check:
+ * </p>
+ * <pre>
+ * &lt;module name="AvoidNestedBlocks"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() {
+ *   int myInteger = 0;
+ *   {                      // violation
```

Comment:
good example , and please use whole example as-is in seconds code block.
https://github.com/checkstyle/checkstyle/wiki/Good-practice-on-how-to-update-examples see block "ATTENTION" for a reason.

Reply:
Done, now both examples are almost same.